srml/contract/Cargo.toml

```diff
@@ -28,6 +29,9 @@ balances = { package = "srml-balances", path = "../balances" }
 
 [features]
 default = ["std"]
+core = [
+	"wasmi-validation/core",
```

Comment:
What is core doing?

Reply:
`core` is actually for cases where `no-default-features` is not enough. `wasmi-validation` uses it for depending on hashbrown in `no_std` mode.

So it is meant that this crate will be compiled with `core` feature if compiled to the `no_std` environment.